src/react/PageRoute.jsx

```diff
@@ -22,7 +22,7 @@ export default function PageRoute(props) {
     if (match) {
       sendPageEvent();
     }
-  }, [match]);
+  }, [JSON.stringify(match)]);
```

Comment:
We are currently logging duplicate page events, so adding `JSON.stringify` here will make sure this effect only runs when something within `match` changes, not when the object reference changes.